airbyte-webapp/src/locales/en.json

```diff
@@ -59,7 +59,7 @@
   "form.credentials_json": "Credentials JSON",
   "form.disable_collection": "Disable Collection",
   "form.replication_method": "Replication Method",
-  "form.validationError": "The received form did not pass validation",
```

Comment:
does any validation happen on the frontend? is it possible that we could actually have a form not pass validation? if it's not possible now that's fine and the backend can do it. we will then need to return different errors on the backend whether it's a validation error or an actual error.

Reply:
As for validation we currently validate only for required. Probably, if validation schema is provided from backend, we can also support validation on the frontend for min|max, regex and etc.

We return different types of errors based on status returned from the backend.

Reply:
great!